relay-log/src/setup.rs

```diff
@@ -173,6 +173,9 @@ pub struct SentryConfig {
     /// Sets the environment for this service.
     pub environment: Option<Cow<'static, str>>,
 
+    /// Sets the server name for this service.
+    pub server_name: Option<Cow<'static, str>>,
```

Comment:
Can the SDK load the server name from an environment variable?

We should also make this env var overridable, especially for k8s deployments this is the only way to have a server name per pod.